server/circleci.go

```diff
@@ -111,12 +111,13 @@ func (s *Server) requestEETriggering(ctx context.Context, pr *model.PullRequest,
 
 func (s *Server) triggerEnterprisePipeline(ctx context.Context, pr *model.PullRequest, info *EETriggerInfo) (*circleci.Pipeline, error) {
 	params := map[string]interface{}{
-		"tbs_sha":           pr.Sha,
-		"tbs_pr":            strconv.Itoa(pr.Number),
-		"tbs_server_owner":  info.ServerOwner,
-		"tbs_server_branch": info.ServerBranch,
-		"tbs_webapp_owner":  info.WebappOwner,
-		"tbs_webapp_branch": info.WebappBranch,
+		"tbs_sha":                  pr.Sha,
+		"tbs_pr":                   strconv.Itoa(pr.Number),
+		"tbs_server_owner":         info.ServerOwner,
+		"tbs_server_branch":        info.ServerBranch,
+		"tbs_server_target_branch": info.BaseBranch,
```

Comment:
The parameter in the [PR](https://github.com/mattermost/enterprise/pull/1362/files) is `target-branch`. Were you planning to rename that again?

Reply:
This will be passed by `<< pipeline.parameters.tbs_server_branch >>` once provided. The fallback will be `<< parameters.target-branch >>` so that we can still have a value if the job is not triggered by a server PR.